server/check_output_directory.js

```diff
@@ -28,16 +29,17 @@ async function get_error(directory) {
         "Check the permissions of the directory, and if needed change them so that " +
         `user with UID ${uid} has access to them. This can be achieved by running the command: chown ${uid}:${gid} on the directory`;
     } finally {
-      return err_msg;
+      console.log(err_msg);
```

Comment:
Wow! This changes the behavior ! 

Please avoid introducing changes like that before discussing them !

If you do feel some behavior change is required, let's discuss it together first, and add tests for it.

Reply:
For above I am getting lint error: Unsafe usage of ReturnStatement.
Explaination:
 In JavaScript, using `return` in a `finally` block can lead to unexpected behavior, and it's generally considered unsafe. The `finally` block is designed for code that should be executed regardless of whether an exception is thrown or caught. Placing a `return` statement in a finally block can interfere with the intended flow.

Reply:
Well, you cannot just remove the return statement. This changes the behavior of the function. In this case, the return was intended. You can refactor it, but not change the behavior. Even less when you are not adding tests.